### PR TITLE
Update/remove auth0 dependencies

### PR DESCRIPTION
**Pull request by amychen1776**
## Description & motivation
This removes the mention of Auth0 as an optional dependency for dbt Cloud.

### CODE REVIEW

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/sso-overview.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ dbt Cloud supports JIT (Just-in-Time) provisioning and IdP-initiated login. You
 
 The diagram below explains the basic process by which users are provisioned in dbt Cloud upon logging in with SSO.
 
-![SSO-diagram](https://user-images.githubusercontent.com/46451573/84216940-e0b60e80-aa98-11ea-91a2-4bffbfffe69e.png)
+<Lightbox src="/img/sso_overview.png" title="SSO-diagram" />
+
+![SSO-diagram](static/img/sso_overview.png)
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/sso-overview.md]: i think we do not want this image here, right?

**Review by drewbanin — CHANGES_REQUESTED** [review]
One changed requested, and I think you might need to merge the latest commits into this branch. Pending that, looks good to me :)

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/sso-overview.md (hunk 1 above)
Comment (amychen1776) [reply website/docs/docs/dbt-cloud/dbt-cloud-enterprise/sso-overview.md]: @drewbanin This is actually a revised version of the SSO diagram, without Auth0 in the mix. :)

Comment (drewbanin) [reply website/docs/docs/dbt-cloud/dbt-cloud-enterprise/sso-overview.md]: yeah - it's a great diagram! But we can just keep the `Lightbox` and remove the `![SSO-diagram]` image embed which is not rendering on the page.

Screen Shot 2020-10-20 at 8 34 10 AM

I don't think we need/want both of these image embeds here.....

Comment (amychen1776) [reply website/docs/docs/dbt-cloud/dbt-cloud-enterprise/sso-overview.md]: Ahhhh I see what you mean. That was an accident 😓 . Fixing that now :)

File: website/docs/docs/dbt-cloud/on-premises/dependencies.md [hunk 2]
```diff
@@ -65,9 +65,13 @@ In order to perform basic maintenance and license checking, the following outbou
 
 - `api.replicated.com`: This endpoint services the license sync check and is used to pull down yaml files for app upgrades.
 
+<<<<<<< HEAD:website/docs/docs/dbt-cloud/on-premises/architecture.md
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/on-premises/dependencies.md]: just make sure to fix this merge artifact

**Pull request closed without merging**